bentoml/cli/__init__.py

```diff
@@ -91,6 +92,8 @@ def run(ctx, api_name, archive_path=archive_path):
                 )
             )
 
+        track_cli(api_name, model_service, ctx.args)
```

Comment:
change `api_name` here to just run, we can log api_name separately...  api_name variable is user-defined and we can't easily use that

Reply:
we should able to have access to api_name at this point. But I think you are right now change it to `run` to standardize events.

Reply:
i mean can’t easily use that for analytics

Reply:
done